src/api/l_audio.c

```diff
@@ -48,15 +49,16 @@ static int l_lovrAudioSetVolume(lua_State* L) {
 static int l_lovrAudioNewSource(lua_State* L) {
   Source* source = NULL;
   SoundData* soundData = luax_totype(L, 1, SoundData);
+  bool spatial = lua_isboolean(L, 2) ? lua_toboolean(L, 2) : true;
```

Comment:
Maybe this should be in an options table.  It would make it more clear when the spatial property is being set:

```lua
lovr.audio.newSource(sound, false) -- Hmm, what is that boolean for
lovr.audio.newSource(sound, { spatial = false }) -- More clear
```

and make it easier to add more configuration to Source objects without making breaking changes to the API.

I think it's okay to still have the bool in the C API, some sort of SourceInfo struct could be added there later if needed.

Reply:
that's a MUCH better API! I also prefer keyword-style APIs and clear parameters! (I'm an old ObjC programmer, after all :P) will fix!